vita3k/emuenv/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(
+	emuenv
+	INTERFACE
+	include/emuenv/app_util.h
+	include/emuenv/state.h
+	include/emuenv/window.h
+)
+
+target_include_directories(emuenv INTERFACE include ${PSVPFSPARSER_INCLUDE_DIR})
+target_link_libraries(emuenv INTERFACE psvpfsparser app audio config ctrl dialog display ime io kernel lang miniz net ngs nids np renderer sdl2 touch gdbstub codec package)
+target_link_libraries(emuenv INTERFACE elfio::elfio FAT16 vita-toolchain)
```

Comment:
You should be able to remove some of these librairies which were used by the code in package now (for sure elfio, fat16 and vita-toolchain, and psvpfsparser,sdl2 too I think).

Reply:
Just confirmed it could be done so I've applied the changes to the CMake file.